observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -36,6 +36,11 @@ Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
 By default, in a newly installed {product-title} system, the monitoring `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
 ====
 +
+[IMPORTANT]
+====
+In multi-node clusters, you must configure persistent storage for Prometheus, Alertmanager, and Thanos Ruler to ensure high availability.
+====
```

Comment:
It is best to avoid stacking admonitions. Could that note be pulled into the content above or could you use an unordered list in the important admonition instead?

This is not a blocker for merge.